topeft/modules/corrections.py

```diff
@@ -311,6 +335,93 @@ def ApplyTES(year, Taus, isData):
     dmFlag = ((Taus.decayMode==0) | (Taus.decayMode==1) | (Taus.decayMode==10) | (Taus.decayMode==11))
     whereFlag = kinFlag & dmFlag #((pt>20) & (pt<205) & (gen==5) & (dm==0 | dm==1 | dm==10 | dm==11))
     tes = np.where(whereFlag, SFevaluator['TauTES_{year}'.format(year=year)](dm,pt), 1)
+
+    ## Correction-lib implementation - MUST BE TESTED WHEN TAU IN THE MASTER BRANCH PROCESSOR
+    padded_Taus = ak.pad_none(Taus,1)
+    padded_Taus = ak.with_name(padded_Taus, "TauCandidate")
+
+    clib_year = clib_year_map[year]
+    json_path = topcoffea_path(f"data/POG/TAU/{clib_year}/tau.json.gz")
+    ceval = correctionlib.CorrectionSet.from_file(json_path)
+
+    pt  = padded_Taus.pt
+    dm  = padded_Taus.decayMode
+    wp  = padded_Taus.idDeepTau2017v2p1VSjet
+    eta = padded_Taus.eta
+    gen = padded_Taus.genPartFlav
+    mass= padded_Taus.mass
+
+    arg_tau = ["pt", "eta", "decayMode", "genPartFlav"]
+    pt_mask_flat = ak.flatten((pt>0) & (pt<1000))
+
+    ## Correction-lib implementation - MUST BE TESTED WHEN TAU IN THE MASTER BRANCH PROCESSOR
+    DeepTaus = [
```

Comment:
Can we change `DeepTaus` to `deep_taus` in order to keep the styling consistent?